docs/Query.md

```diff
@@ -15,7 +15,6 @@ The Query command can be called by either the cloud or INBC.  It will provide at
 
 | Attribute     | Description                                         | 
 |:--------------|:----------------------------------------------------|
-| is_flashless  | True if plug-in card is flashless; otherwise, False |
 | manufacturer  | Hardware manufacturer                               |
 | platform_type | Type of plug-in card.  TBH or KMB                   |
```

Comment:
Maybe we should remove platform_type in this doc as it's for the type of plug-in card.

Reply:
good point.  removed.